src/ui/profile_preferences.ui

```diff
@@ -134,6 +134,28 @@
         </widget>
        </item>
        <item row="1" column="0">
+        <widget class="QGroupBox" name="groupBox_encoding">
+         <property name="enabled">
+          <bool>true</bool>
+         </property>
+         <property name="title">
+          <string>Character Encoding</string>
```

Comment:
Should be `Character encoding`, but how about `Text encoding` for something less technical? Not that `encoding` isn't - but there's no way around that.